src/content/applications/RespondusLockDownBrowser.md

```diff
@@ -0,0 +1,8 @@
+---
+name: "Respondusâ€‹ LockDown Browser"
```

Comment:
there are still some junk characters in the name here. Did you find the source of these?